src/@components/JoinPage/common/SuccessBtn/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { St } from "./style";
+
+interface ButtonContentsProps {
+  name: string;
+}
+
+export default function SuccessBtn({ name }: ButtonContentsProps) {
+  return <St.SuccessButton>{name}</St.SuccessButton>;
+}
```

Comment:
✔ props 컨벤션 지켜주세요!
아래 suggestion 그대로 올리셔도 좋을 것 같습니다

```suggestion
export default function SuccessBtn(props: ButtonContentsProps) {
  const { name } = props;
  
  return <St.SuccessButton>{name}</St.SuccessButton>;
}
```

Reply:
에구궁.. 컨벤션이었구나 제송함미다...ㅇㅅㅇ